tests/utils/qemu.spec.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @license
+ * Copyright 2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+
+describe('resin-multibuild consistency', function() {
+	it('should use the same values for selected constants', async () => {
+		const { QEMU_BIN_NAME: MQEMU_BIN_NAME } = await import('resin-multibuild');
+		const { QEMU_BIN_NAME } = await import('../../build/utils/qemu-ts');
+		expect(QEMU_BIN_NAME).to.equal(MQEMU_BIN_NAME);
```

Comment:
"Peace of mind" test as both `balena-cli` and `resin-multibuild` have `QEMU_BIN_NAME` constants that should be equal. I thought of defining `QEMU_BIN_NAME` in `resin-multibuild` only and then importing it in `balena-cli`, however this would require either importing at module level / top of file (which hurts CLI start-up time), or importing `resin-multibuild` in the body of several functions, which felt... inelegant.